scripts/run_lighthouse_tests.py

```diff
@@ -190,29 +206,31 @@ def main(args=None):
     """Runs lighthouse checks and deletes reports."""
     parsed_args = _PARSER.parse_args(args=args)
 
-    if not (parsed_args.accessibility or parsed_args.performance):
-        _PARSER.error(
-            'Must pass either --accessibility or --performance flag')
+    if parsed_args.mode == 'accessibility':
+        lighthouse_mode = LIGHTHOUSE_MODE_ACCESSIBILITY
+        server_mode = SERVER_MODE_DEV
+    elif parsed_args.mode == 'performance':
+        lighthouse_mode = LIGHTHOUSE_MODE_PERFORMANCE
+        server_mode = SERVER_MODE_PROD
```

Comment:
else raise.

That said -- you have a dict, so you can do "try"/"except KeyError" instead.

Reply:
Sorry, scratch that, I misread this. Do an "else raise" instead (ignore the second part).